### PR TITLE
Add Openai to llms testing

### PR DESCRIPTION
**Pull request by Jeadie**
## 🗣 Description
 - Add OpenAI to `llms` package testing.
 - Fix `.github/workflows/integration_llms.yml` by adding `dotenvy`

### CODE REVIEW

File: crates/llms/tests/llms/snapshots/integration__llms__tool_use_openai_finish_reason.snap [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+source: crates/llms/tests/llms/mod.rs
+expression: "serde_json::to_string_pretty(&resp_ptr).expect(\"Failed to serialize snapshot\")"
+snapshot_kind: text
+---
```
Comment (Jeadie) [inline crates/llms/tests/llms/snapshots/integration__llms__tool_use_openai_finish_reason.snap:5]: Turns out OpenAI isn't OpenAI compatible either :(

**Review by sgrebnov — APPROVED** [review]

File: .github/workflows/integration_llms.yml [hunk 2]
```diff
@@ -73,15 +73,8 @@ jobs:
 
       - name: Set up Anthropic API Key
         run: |
-          echo 'SPICE_ANTHROPIC_API_KEY="${{ secrets.SPICE_SECRET_ANTHROPIC_API_KEY }}"' > .env
+          echo 'SPICE_ANTHROPIC_API_KEY="${{ secrets.SPICE_SECRET_ANTHROPIC_API_KEY }}"\nSPICE_OPENAI_API_KEY="${{ secrets.SPICE_SECRET_OPENAI_API_KEY }}"' > .env
 
       - name: Run integration test
-        env:
-          SPICE_SECRET_ANTHROPIC_API_KEY: ${{ secrets.SPICE_SECRET_ANTHROPIC_API_KEY }}
         run: |
```
Comment (sgrebnov) [inline .github/workflows/integration_llms.yml:79]: Can we create separate jobs for OpenAI and Anthropic (could be separate PR)?
